### PR TITLE
fix `The 'noscript' tag is wrongly processing in innerHTML wrapper` (close #987)

### LINKED ISSUE #987: The 'noscript' tag is wrongly processing in innerHTML wrapper
```javascript
var div = document.createElement('div');

div.innerHTML = '<noscript><div></noscript> <span></span> <noscript></div></noscript>';
console.log(div.innerHTML);
// <noscript>&lt;div&gt;</noscript> <span></span> <noscript>&lt;/div&gt;</noscript>

__set$(div, "innerHTML", '<noscript><div></noscript> <span></span> <noscript></div></noscript>');
console.log(div.innerHTML);
// <noscript>&lt;div&gt; &lt;span&gt;&lt;/span&gt; &lt;noscript&gt;</noscript>

### PR DESCRIPTION
**Pull request by LavrovArtem**
/cc @churkin @miherlosev

### CODE REVIEW

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

File: src/client/utils/html.js [hunk 1]
```diff
@@ -10,17 +10,21 @@ import { hasIsNotClosedFlag } from '../sandbox/node/document/writer';
...
 import INTERNAL_PROPS from '../../processing/dom/internal-properties';
 
-const FAKE_TAG_NAME_PREFIX  = 'fake_tag_name_';
-const FAKE_DOCTYPE_TAG_NAME = 'hammerhead_fake_doctype';
-const FAKE_TAG_NAME_RE      = new RegExp('(<\\/?)' + FAKE_TAG_NAME_PREFIX, 'ig');
-const WRAP_TAGS_RE          = /(<\/?)(html|head|body|table|tbody|tfoot|thead|tr|td|th|caption|colgroup)((?:\s[^>]*)?>)/ig;
-const WRAP_TAGS_TEMPLATE    = `$1${ FAKE_TAG_NAME_PREFIX }$2$3`;
-const WRAP_COL_TAG_RE       = /<(\/?(col)(?:[^A-Za-z_>][^>]*)?)>/ig;
-const WRAP_COL_TAG_TEMPLATE = `<${ FAKE_TAG_NAME_PREFIX }$2>$1</${ FAKE_TAG_NAME_PREFIX }$2>`;
+const FAKE_TAG_NAME_PREFIX    = 'hh_fake_tag_name_';
```
Comment (miherlosev) [inline src/client/utils/html.js]: 'hh_fake_tag_name_' - remove last underscore

Comment (LavrovArtem) [inline src/client/utils/html.js]: `hh_fake_tag_name_` is prefix. For example:
`<table> -> <hh_fake_tag_name_table>`

Comment (miherlosev) [inline src/client/utils/html.js]: Ok

**Comment by LavrovArtem** [comment]
ping @churkin

**Pull request merged**